homeassistant/components/lyric/__init__.py

```diff
@@ -201,11 +201,25 @@ def device_info(self) -> DeviceInfo:
             identifiers={
                 (
                     f"{dr.CONNECTION_NETWORK_MAC}_room_accessory",
-                    f"{self._mac_id}_room{self._room.id}_accessory{self._accessory.id}",
+                    f"{self._mac_id}_room{self._room_id}_accessory{self._accessory_id}",
                 )
             },
             manufacturer="Honeywell",
             model="RCHTSENSOR",
-            name=f"{self._room.roomName} Sensor",
+            name=f"{self.room.roomName} Sensor",
             via_device=(dr.CONNECTION_NETWORK_MAC, self._mac_id),
         )
+
+    @property
+    def room(self) -> LyricRoom:
+        """Get the Lyric Device."""
+        return self.coordinator.data.rooms_dict[self._mac_id][self._room_id]
+
+    @property
+    def accessory(self) -> LyricAccessories:
```

Comment:
The `LyricAccessories` object is, in fact, a single accessory.  This is misnamed in the dependent library `aiolyric` that this object originates from.

Reply:
Feel free to make a PR in the package and we can deploy that first

Reply:
Yeah, I was gonna fix it if/when the library needed updated again (as it was my mistake in the first place), but didn't feel it warranted intimidate action since it was just semantic. 

I do have a couple PRs ready to go for after this bug is fixed (adding a sensor and binary_sensor respectively).  As this is code readability issue, I can add a PR fixing this as well (or fix it prior if we think that'd be better).

Reply:
For other reviewers FYI, @timmo001 and I fixed this in [version 2.0.1](https://github.com/timmo001/aiolyric/releases/tag/2.0.1).  

If anyone would like this fixed before this PR is merged, let me know.  Otherwise, I will open a PR to bump the dependency version as a fast-follow once this PR is merged.